Packs/VMware/Integrations/VMware/VMware.py

```diff
@@ -0,0 +1,415 @@
+# pylint: disable=no-member
+# pylint: disable=no-name-in-module
+
+import ssl
+import sys
+from datetime import datetime
+
+import demistomock as demisto  # noqa: F401
+from CommonServerPython import *  # noqa: F401
+from cStringIO import StringIO
+from pyVim.connect import Disconnect, SmartConnect
+from pyVim.task import WaitForTask
+from pyVmomi import vim, vmodl
```

Comment:
are all these imports sorted and needed? if not, please tend to it.

Reply:
All are needed (I removed the unnecessary ones).
What's our best practice regarding sorting imports?  This is how it sorted on the original integration.

Reply:
i like using: https://pypi.org/project/isort/
isort {path_to_py_file}